_includes/tutorials/ksql-concatenation-columns/ksql/markup/dev/create-financial-transaction-stream.adoc

```diff
@@ -0,0 +1,8 @@
+The first thing we do is to create a stream `ACTIVITY_STREAM` based off the topic `stock_purchases`.  The data contained in the topic is just plain JSON, with no special structure.
```

Comment:
1. Is the sequencing right?  The topic hasn't been created yet (that's in the next step).
2. What does "no special structure" mean?  Is there a more technical term that can be used?  (schemaless?)